lib/raster/mask_info.c

```diff
@@ -70,3 +70,13 @@ int Rast__mask_info(char *name, char *mapset)
 
     return 1;
 }
+
+/**
+ * @brief Check presence of 2D raster mask
+ *
+ * @return true if mask is present, false otherwise
+ */
+bool Rast_mask_is_present()
```

Comment:
```suggestion
bool Rast_mask_is_present(void)
```

Is this the way to fix the ~no prototype deprecated in all versions of C..?

Reply:
Added void and the missing prototype.